tests/interactive.fish

```diff
@@ -22,11 +22,13 @@ cd (dirname (status -f))
 set -gx TERM xterm
 set -e ITERM_PROFILE
 
-# Test files specified on commandline, or all *.expect files
+# Test files specified on commandline, or all *.expect files.
 if set -q argv[1]
-    set files_to_test $argv.expect
+    set expect_files_to_test $argv.expect*
```

Comment:
This will break because it matches legacy `*.expect.err` and `*.expect.out` paths.

I think we should drop this argument handling anyway because it should made be easier to run individual tests in the same way the build system would run them.
Merely running `fish interactive.fish bind` is usually wrong, or I just don't understand how to use it. It does not know about the build system, so it does not copy changed tests, `*.err` or `*.out` files, and it does not rebuild fish. Not sure in which scenarios this would be desired.

What I usually do is edit `interactive.fish` to hardcode `files_to_test`, and then run `make test_interactive`.

Maybe we can pass it as env var instead:
`INTERACTIVE_TESTS=./tests/pexpects/*.py ninja test_interactive`

Reply:
Fixed the globbing; I will leave making it easier to run specific tests for another day.